api/system/v1alpha1/secret.proto

```diff
@@ -15,6 +15,7 @@ message Secret {
   option (kuma.mesh.resource).package = "system";
   option (kuma.mesh.resource).skip_kubernetes_wrappers = true;
   option (kuma.mesh.resource).kds.send_to_zone = true;
+  option (kuma.mesh.resource).kds.send_to_global = false;
```

Comment:
Does this supplement make impact? I don't see any place call it.
Or is it just a standard supplement to the structure?

Reply:
It makes no impact since the default is `false`. I wanted to make it explicit